reconcile/ocm_clusters.py

```diff
@@ -95,6 +131,15 @@ def run(dry_run, gitlab_project_id=None, thread_pool_size=10):
                 desired_spec['spec'].pop(k, None)
 
             if current_spec != desired_spec:
+                # check if cluster update is valid
+                update_spec, err = get_cluster_update_spec(
```

Comment:
This is better served with a `try-except`.

Reply:
"better" is subjective :)